mhkit/dolfyn/io/nortek_defs.py

```diff
@@ -340,3 +340,144 @@ def sci_func(self, data):
                     standard_name='signal_intensity_from_multibeam_acoustic_doppler_velocity_sensor_in_sea_water',
                     ),
 }
+
+waves_hdrdata = {
+    'time_alt': _VarAtts(dims=[],
+                         dtype=np.float64,
+                         group='coords',
+                         default_val=nan,
+                         units='seconds since 1970-01-01 00:00:00 UTC',
+                         long_name='Time',
+                         standard_name='time',
+                         ),
```

Comment:
should this have a `factor` key?

Reply:
No, the factor key here is for any variables that are stored in binary in something other than their native units. Time is stored in a binary format that converts directly to "seconds since 1970"